cmd/minishift/cmd/addon/addon_enable.go

```diff
@@ -38,7 +38,7 @@ var addonsEnableCmd = &cobra.Command{
 }
 
 func init() {
-	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of the add-on when it is applied.")
+	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of the add-on. Determines the order in which to apply the add-on.")
```

Comment:
Because it is not quite correct. This is does not determine the order of this single add-on. To determine an order I need to take more than one thing into account.